### DISCUSSION TITLE
Metadata operation metrics for FlexGroup volumes

### DISCUSSION THREAD

**Discussion by charz** [NetApp/harvest#764] [Q&A] [answered]
Do we have metadata operation metrics of volume that can be exposed by harvest? Does this already support?

**Comment by rahulguptajss** [comment]
@charz Could you let us know which all fields are you looking for or if you know of zapi which provides this information?

**Reply by charz** [reply]
@rahulguptajss Thanks for pointing that, I'm not aware the zapi provide this information. Do you mean something that I can enable or change settings on these yaml configure in https://github.com/NetApp/harvest/blob/main/conf/zapiperf/default.yaml ?

I do enable these to collect more details for Volume, but I'm looking for metrics for meta-data operations (GETATTR, READDIRPLUS, ACCESS, LOOKUP, etc) in a volume. I checked the metrics I got but nothing related to metadata for a volume.

```
      # Uncomment to collect workload/QOS counters
      Workload:               workload.yaml
      WorkloadDetail:         workload_detail.yaml
      WorkloadVolume:         workload_volume.yaml
      WorkloadDetailVolume:   workload_detail_volume.yaml
```

**Reply by ruanruijuan** [reply]
@charz https://github.com/NetApp/harvest/blob/main/conf/zapiperf/cdot/9.8.0/nfsv3.yaml (and nfsv4.yaml) has these types of perf metrics, but at SVM level. I'm not aware of such metrics at volume level. Is this what you are looking for?

**Reply by charz** [reply]
@ruanruijuan Yes, I'm looking for these metrics. Sounds like we only have SVM level metrics. Thanks!

**Comment by ruanruijuan** [comment, marked as answer]
I'm moving this issue to "Discussion".
